charts/kubefed/charts/controllermanager/templates/roles.yaml

```diff
@@ -81,3 +81,34 @@ rules:
   - secrets
   verbs:
   - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    api: federation
```

Comment:
s/federation/kubefed/

Reply:
Fixed. But the others not changed in this PR will also need to be fixed in another PR.